File: Python/01 - Introduction to Programming/Task17/disappear.py
#CJ Patel
#Task17 - disappear.py
#------------------------------------------------------------#
#request input from user
original = input("Please enter a value for a string variable: ")
characters = input("Which characters would you like to make disappear? ") #to be entered without commas or spaces
#characters = characters.replace(",","") ; if characters are seperated by commas
disappear = original

#remove specified characters
for i in range (0,len(characters)):
  disappear = disappear.replace(characters[i],"")

print(disappear)

File: Ruby/01 - Introduction to Programming/Task17/disappear.rb
#Task17 - disappear.rb
#------------------------------------------------------------#
#request input from user
print "Please enter a value for a string variable: "
original = gets.chomp()
print "Which characters would you like to make disappear? "
characters = gets.chomp() #to be entered without commas or spaces
#characters = characters.gsub(",","") ; if characters are seperated by commas
disappear = original

#remove specified characters
for i in Range.new(0,characters.length)
  disappear = disappear.gsub(characters[i],"")
end
      
puts disappear